src/packages/@ncigdc/modern_components/CohortComparison/FacetTable.js

```diff
@@ -1,184 +1,191 @@
-                                    value: [`set_id:${set1}`],
-                                  },
-                                },
-                                {
-                                  op: 'in',
-                                  content: {
-                                    field,
-                                    value: [set1_bucket.key],
-                                  },
-                                },
-                              ],
-                            },
-                          }}
-                        >
-                          {(set1_bucket.doc_count || 0).toLocaleString()}
-                        </ExploreLink>}
-                  </Td>
-                  <Td style={{ textAlign: 'right' }}>
-                    {((set1_bucket.doc_count || 0) /
-                      result1.hits.total *
-                      100).toFixed(0)}%
-                  </Td>
-                  <Td style={{ textAlign: 'right' }}>
-                    {(set2_bucket.doc_count || 0) === 0
-                      ? 0
-                      : <ExploreLink
-                          query={{
-                            searchTableTab: 'cases',
-                            filters: {
-                              op: 'and',
-                              content: [
-                                {
-                                  op: 'in',
-                                  content: {
-                                    field: 'cases.case_id',
-                                    value: [`set_id:${set2}`],
-                                  },
-                                },
-                                {
-                                  op: 'in',
-                                  content: {
-                                    field,
-                                    value: [set2_bucket.key],
-                                  },
-                                },
-                              ],
-                            },
-                          }}
-                        >
-                          {(set2_bucket.doc_count || 0).toLocaleString()}
-                        </ExploreLink>}
-                  </Td>
-                  <Td style={{ textAlign: 'right' }}>
-                    {((set2_bucket.doc_count || 0) /
-                      result2.hits.total *
-                      100).toFixed(0)}%
-                  </Td>
-                </Tr>
-              );
-            })}
-          </tbody>
-        }
-      />
-      <div style={{ textAlign: 'right' }}>
-        {buckets1.length === 2 &&
-          <Pvalue
-            data={[
-              buckets1.map(x => x.doc_count),
-              buckets2.map(x => x.doc_count),
-            ]}
-          />}
+    return (
+      <div>
+        <h2>{heading}</h2>
+        <BarChart
+          data1={tableData.map(({ value, casesS1 }) => ({
+            label: truncate(value, { length: 15 }),
+            value: casesS1,
```

Comment:
for example:

label: value,
value: not_value

is a touch confusing.

Reply:
Ahh I see. Good point. I'll see what I can come up with